refs/biblio.json

```diff
@@ -3330,6 +3380,41 @@
     "SSML10": {
         "aliasOf": "speech-synthesis"
     },
+    "SSNX": {
```

Comment:
This seems to be an article that describes the SSNX ontology. Shouldn't that rather target the [SSNX ontology itself](https://www.w3.org/2005/Incubator/ssn/ssnx/ssn)?